services/net/cpnews/Xml/Atom/AtomFeed.cs

```diff
@@ -0,0 +1,144 @@
+using System.ServiceModel.Syndication;
+using System.Xml;
+using System.Xml.Linq;
+using System.Xml.Serialization;
+
+namespace TNO.Services.CPNews.Xml;
+
+/// <summary>
+/// AtomFeed class, provides an object to read and write Atom syndication feed.
+/// </summary>
+[XmlRoot("atom")]
+public class AtomFeed
```

Comment:
The duplication of the Xml folder isn't good.  Need to centralize this in the Services library.

Reply:
yes